packages/suite/src/reducers/wallet/coinmarketReducer.ts

```diff
@@ -35,7 +36,8 @@ type CommonTrade = {
 
 export type TradeBuy = CommonTrade & { tradeType: 'buy'; data: BuyTrade };
 export type TradeExchange = CommonTrade & { tradeType: 'exchange'; data: ExchangeTrade };
-export type Trade = TradeExchange | TradeBuy;
+export type TradeSpend = CommonTrade & { tradeType: 'spend'; data: SpendTrade };
+export type Trade = TradeExchange | TradeBuy | TradeSpend;
```

Comment:
commented above, consider move all types to common file

Reply:
done